wc/Main.hs

```diff
@@ -0,0 +1,59 @@
+module Main where
+
+import Wc
+import TryingSingletons
+import Options.Applicative
+import qualified Data.Set as Set
+import Data.Maybe
+
+main :: IO ()
+main = execParser (info (helper <*> parseOpts) myInfoMod) >>= print
```

Comment:
right now this just prints out what options it parsed, will need to change it to call the function to do the counting on each file argument, and then finally print out the results